### PR TITLE
*: add more contents about supported versions and upgrade methods

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/15079
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: smooth-upgrade-tidb.md [hunk 1]
```diff
@@ -20,16 +33,31 @@ Before the smooth upgrade feature is introduced, there are the following limitat
...
+#### Other upgrade methods
+
+You can use the following steps to upgrade TiDB manually or by using a script:
+
+1. Send the HTTP upgrade start request to any TiDB node in the cluster: `curl -X POST http://{TiDBIP}:10080/upgrade/start`.
+   * The TiDB cluster enters the **Upgrading** state.
+   * The DDL operations to be performed are paused.
+
+2. Replace the TiDB binary and perform a rolling upgrade. This process is the same as the original upgrade process.
+    * Perform system DDL operations during the upgrade process.
```
Comment (ran-huang) [inline smooth-upgrade-tidb.md:57]:
```suggestion
    * The system DDL operations are performed during the upgrade process.
```

**Review by ran-huang — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: smooth-upgrade-tidb.md [hunk 2]
```diff
@@ -11,25 +11,67 @@ summary: This document introduces the smooth upgrade feature of TiDB, which supp
```
Comment (zimulala) [inline smooth-upgrade-tidb.md:11]: We need to remove line8-10.

File: smooth-upgrade-tidb.md [hunk 3]
```diff
@@ -54,3 +96,7 @@ When using the smooth upgrade feature, note the following limitations.
     * BR: BR might replicate the paused DDL jobs to TiDB. The paused DDL jobs cannot be automatically resumed, which might cause the DDL jobs to be stuck later.
 
     * DM and TiCDC: If you use DM or TiCDC to import SQL statements to TiDB during the upgrade process, and if one of the SQL statements contains DDL operations, the import operation is blocked and undefined errors might occur.
+
+### Limitation on plugins
+
+The plugins installed in TiDB might contain DDL operations. However, during the upgrade, if the DDL operations in the plugins are not performed on system tables, the upgrade might fail.
```
Comment (zimulala) [inline smooth-upgrade-tidb.md:102]:
```suggestion
The plugins installed in TiDB might contain DDL operations. However, during the upgrade, if the DDL operations in the plugins are performed on non-system tables, the upgrade might fail.
```

File: smooth-upgrade-tidb.md [hunk 4]
```diff
@@ -11,25 +11,67 @@ summary: This document introduces the smooth upgrade feature of TiDB, which supp
...
+
+Currently, this feature is not supported. It will be supported as soon as possible.
+
+#### Other upgrade methods
+
+You can take the following steps to upgrade TiDB manually or by using a script:
+
+1. Send the HTTP upgrade start request to any TiDB node in the cluster: `curl -X POST http://{TiDBIP}:10080/upgrade/start`.
+   * The TiDB cluster enters the **Upgrading** state.
+   * The DDL operations to be performed are paused.
```
Comment (zimulala) [inline smooth-upgrade-tidb.md:68]:
```suggestion
   * The user DDL operations to be performed are paused.
```

File: smooth-upgrade-tidb.md (hunk 2 above)
Comment (hfxsd) [reply smooth-upgrade-tidb.md:11]: done

**Review by zimulala — APPROVED** [review]

**Pull request merged**